src/Config.js

```diff
@@ -77,7 +80,23 @@ class Config {
     return mergedConfig;
   };
 
+  onConfigChange = (callback) => {
+    const watcher = chokidar.watch([DEFAULT_CONFIG_PATH, CONFIG_PATH]);
+    watcher.on('change', async () => {
+      let tempConfig = this._loadConfig();
+      // Check if onvif device config changed. Only this change observe
+      if (tempConfig?.onvif !== undefined && this._config?.onvif !== undefined) {
+        let temp = await JSON.stringify(tempConfig.onvif),
+            current =  await JSON.stringify(this._config.onvif);
+        if (temp!== current) {
```

Comment:
I suggest we use `lodash.isEqual` here to check for deep equality as the following is not equal:
![image](https://user-images.githubusercontent.com/655800/103394822-b7be9f00-4af8-11eb-861d-e6ef49a568a2.png)